contracts/transmuter/src/incentive_pool.rs

```diff
@@ -78,10 +78,10 @@ impl IncentivePool {
     /// Credit normalized incentive amount to user (when they should receive incentives)
     /// Validates that total credits don't exceed normalized pool value and caps if necessary
     /// Returns the actual amount credited (may be less than requested if capped)
-    pub fn credit_incentive_to_user(
+    pub fn credit_incentive(
```

Comment:
ACK the rename here, I will have to review the logic of this function separately